spec/requests/internal/profile_fields_spec.rb

```diff
@@ -0,0 +1,95 @@
+require "rails_helper"
+
+RSpec.describe "/internal/profile_fields", type: :request do
+  include ActiveJob::TestHelper
```

Comment:
**thought (non-blocking):** I don't think we need this here. Actually we shouldn't need it anywhere anymore, as we're no longer using `ActiveJob`.

Reply:
Removed it from the app and double checked that the specs still pass